CHANGES.md

```diff
@@ -5,6 +5,7 @@ Change Log
 ### 1.0.46
 
 * Fixed an incorrect require (`URIjs` instead of `urijs`).
+* Added Proj4 projections to the location bar. Clicking on the bar switches between lats/longs and projected coordinates. To enable this, set `useProjection` to `true`
```

Comment:
Can you move this up to a new 1.0.47 section while you're making the other changes?